eng/templates/Azure.ServiceTemplate.Template/src/autorest.md

```diff
@@ -0,0 +1,16 @@
+# Azure SDK Code Generation for Data Plane
```

Comment:
There is no `Generated` code under `eng/templates/Azure.ServiceTemplate.Template/src/`? I think it is needed as it will help the partner teams to verify how the generated code would look like.

Reply:
It is SDK project package, the client library has not be generated, and the Generated folder will be generated later.